### ISSUE TITLE
Jupyter Plugin Initialization Failure in Polypheny DB Environment

### ISSUE THREAD

**Issue by Rahulxx01** [polypheny/Polypheny-DB#482]
### Operating System (Polypheny-DB)

Windows

### Polypheny-DB Version

master

### Affected Areas

Something else (please specify in the description!)

### Browser

Chrome

### Bug Description

After running Polypheny DB, the Jupyter Plugin fails to load, preventing the initiation of Polypheny Notebooks. This behaviour was not observed previously when the master branch was updated. Even after deleting the .polypheny folder, the Jupyter plugin remains unresponsive. The Config File indicates that the Docker instance is connected to localhost. All these details are provided in the screenshot below.

<img width="1117" alt="image" src="https://github.com/polypheny/Polypheny-DB/assets/24786147/5fee6c06-bd04-4787-9234-2c6bb49a0866">

<img width="1114" alt="image" src="https://github.com/polypheny/Polypheny-DB/assets/24786147/c05d9034-a376-4185-8fd0-d649c6cb7403">

[v0.9.2-RC1](https://github.com/polypheny/Polypheny-DB/releases/tag/v0.9.2-RC1)

### Reproduction Steps

1. **Start Polypheny DB**.
   
2. **Navigate to Notebooks**.
   
3. **Check for Jupyter Plugin availability**:
   - If the Jupyter Plugin is available, proceed to step 4.
   - If not available, skip steps 4 to 7 and proceed to step 8.

4. **Launch Jupyter server**.
   
5. **Create a new notebook**.
   
6. **Perform tutorials**:
   - Utilize the notebook to follow tutorials or perform desired tasks.

7. **Close the notebook**:
   - Save any changes if necessary.

8. **Stop Jupyter server** (if launched in step 4).
   
9. **Close Polypheny DB**.
   
10. **Rerun the process** as needed.

### Expected Behavior

<img width="632" alt="image" src="https://github.com/polypheny/Polypheny-DB/assets/24786147/876907ac-1d04-4e4e-8b61-9d71bb2564a8">

**Comment by EslamAsHhraf** [comment]
@Rahulxx01 this discussion may help you https://github.com/orgs/polypheny/discussions/477

**Comment by Rahulxx01** [comment]
@EslamAsHhraf I tried deleting the .polypheny file but still the issue is not solved in the new master branch which was updated.

**Comment by Rahulxx01** [comment]
<img width="1116" alt="image" src="https://github.com/polypheny/Polypheny-DB/assets/24786147/1a1511d3-b526-43be-b00d-c35e31af5f43">


Additionally getting this new issue even if I create a notebook I am unable to open it.

**Comment by datomo** [comment]
Hey @Rahulxx01 

Yes, this is a known error. I have added a fix which you should receive automatically if you are not manually building Polyphen-UI.

**Comment by Rahulxx01** [comment]
I am not manually binding Polyphen-UI I am just running Polypheny-DB once the build is complete. @datomo

**Comment by khushi205** [comment]
yes Yes, this is a known error. I am create a Polyphen-UI.
